backend/grpc.go

```diff
@@ -0,0 +1,46 @@
+package backend
+
+import (
+	"context"
+
+	bproto "github.com/grafana/grafana-plugin-sdk-go/genproto/go/grafana_plugin"
+)
+
+type GRPCClient struct {
+	client bproto.BackendPluginClient
+}
+
+// BackendPlugin is the Grafana Backend plugin interface.
+// It corresponds to: grafana.plugin protobuf: BackendPlugin Service | genproto/go/grafana_plugin: BackendPluginClient interface
+type BackendPlugin interface {
+	Check(ctx context.Context, req *bproto.PluginStatusRequest) (*bproto.PluginStatusResponse, error)
+	DataQuery(ctx context.Context, req *bproto.DataQueryRequest) (*bproto.DataQueryResponse, error)
```

Comment:
Does this also need something like:
```
PluginRequest(ctx context.Context, req *bproto.RESTRequest)
```

Reply:
Ya caught this on my own passthrough fixing some comments. Not renamed yet thought.